### PR TITLE
Fix engine setup files.

### PR DESCRIPTION
**Pull request by DelinWorks**

### CODE REVIEW

**Comment by halx99** [comment]
The template cmake needs modify at the same time:
https://github.com/axis-project/axis/blob/dev/templates/cpp-template-default/CMakeLists.txt#L43

**Comment by halx99** [comment]
> The template cmake needs modify at the same time: https://github.com/axis-project/axis/blob/dev/templates/cpp-template-default/CMakeLists.txt#L43
> 
> and lua template cmake script also needs sync

No matter, I will do the cmake scripts replacement

**Comment by DelinWorks** [comment]
I hate github desktop with passion

**Comment by DelinWorks** [comment]
> > The template cmake needs modify at the same time: https://github.com/axis-project/axis/blob/dev/templates/cpp-template-default/CMakeLists.txt#L43
> > and lua template cmake script also needs sync
> 
> No matter, I will do the cmake scripts replacement

Don't stress it @halx99 I'll do it

**Comment by halx99** [comment]
> > > The template cmake needs modify at the same time: https://github.com/axis-project/axis/blob/dev/templates/cpp-template-default/CMakeLists.txt#L43
> > > and lua template cmake script also needs sync
> > 
> > 
> > No matter, I will do the cmake scripts replacement
> 
> Don't stress it @halx99 I'll do it

ok.

File: CMakeLists.txt [hunk 1]
```diff
@@ -4,7 +4,7 @@
 # Copyright (c) 2015-2017 Chukong Technologies Inc.
 # Copyright (c) 2021-2022 Bytedance Inc.
 #
-# https://adxeproject.github.io/
+# https://axisproject.github.io/
```
Comment (halx99) [inline CMakeLists.txt]: should be: axis-project.githib.io

Comment (DelinWorks) [reply CMakeLists.txt]: I've changed it before I don't know why it still says that:

**Comment by DelinWorks** [comment]
the project creation process now works after these commits.

**Comment by halx99** [comment]
> the project creation process now works after these commits. Also, cmake works just fine! I just tested it 👍

But the ci failed, some base cmake files in `axis/cmake/Modules` not modified

**Comment by DelinWorks** [comment]
> > the project creation process now works after these commits. Also, cmake works just fine! I just tested it 👍
> 
> But the ci failed, some base cmake files in `axis/cmake/Modules` not modified

yeah there are many files that have to be changed and there are a lot of them, could you merge this PR just for a checkpoint or do i have to creat another branch?

**Comment by halx99** [comment]
I can help modify them after lunch

**Comment by halx99** [comment]
Now, I'm working on your branch

**Comment by halx99** [comment]
Next step, could simply replace all `https://adxeproject.github.io` to `https://axis-project.github.io` in all sources

**Comment by halx99** [comment]
If the latest ci check succed, I will merge this PR as a checkpoint

**Pull request merged**